### PR TITLE
Update notifications structure to new pattern - part 1

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- The notification pattern was updated, and support for the old version of the pattern is removed in Vanilla 3.x, this PR updates many instances across the project to the new structure - see the upgrade guide: https://vanillaframework.io/docs/migration-guide-to-v3#notifications
- There are notifications on 100+ pages/templates, so there'll be at least one more follow up PR to make it slightly easier to review

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check the files that were changed, and find the pages on the demo, see that the notifications on them are not broken

## Issue / Card

(partially) fixes https://github.com/canonical-web-and-design/web-squad/issues/4854

### CODE REVIEW

**Comment by sowasred2012** [comment]
@jpmartinspt @albertkol @ClementChaumel could one of you take a look at this on /advantage please? This updates the notification pattern structure, and there's a few notifications on the shop, so a QA from someone who knows their way around would be much appreciated.

**Comment by petesfrench** [comment]
For the Kubernetes docs there is a 'File a bug' notification that is the same on every page except for the link to original github file, we could make this into a template.

File: templates/openstack/install.html [hunk 1]
```diff
@@ -121,9 +121,10 @@ <h2>Single-node, multi-node or large-scale cluster?</h2>
     <h2>Single-node OpenStack deployment</h2>
     <p class="u-sv3">These instructions use <a href="https://snapcraft.io/microstack" class="p-link--external">MicroStack</a> &dash; OpenStack in a snap, MicroStack is a pure upstream OpenStack distribution designed for small scale and edge deployments, that can be installed and maintained with a minimal effort.</p>
     <div class="p-notification">
-      <p class="p-notification__response">
-        <span class="p-notification__status">NOTE:</span> MicroStack is in a beta state. We encourage you to test it, give us your <a class="p-link--external" href="https://bugs.launchpad.net/microstack">feedback</a> and <a class="p-link--external" href="https://discourse.juju.is/search?q=microstack">ask questions</a>.
-      </p>
+      <div class="p-notification__content">
+        <span class="p-notification__title">NOTE:</span>
```
Comment (petesfrench) [inline templates/openstack/install.html:125]:
```suggestion
        <span class="p-notification__title">Note:</span>
```
Lower-case to match other instances of notes

File: templates/openstack/install.html [hunk 2]
```diff
@@ -121,9 +121,10 @@ <h2>Single-node, multi-node or large-scale cluster?</h2>
...
+        <p class="p-notification__message">MicroStack is in a beta state. We encourage you to test it, give us your <a class="p-link--external" href="https://bugs.launchpad.net/microstack">feedback</a> and <a class="p-link--external" href="https://discourse.juju.is/search?q=microstack">ask questions</a>.</p>
```
Comment (petesfrench) [inline templates/openstack/install.html:126]: In most cases NOTE: + the message are inline

Comment (sowasred2012) [reply templates/openstack/install.html:126]: Updated

**Review by petesfrench — APPROVED** [review]
Everything except 'advantage', which I didn't look at LGTM :+1:

**Comment by ClementChaumel** [comment]
Everything looked ok on advantage! :+1:

**Pull request merged**